extensions/@shopgate-theme-config/frontend/config-common.js

```diff
@@ -18,12 +18,12 @@ export default {
         show: true,
       },
       '@shopgate/engage/product/EffectivityDates': {
-        showStartDate: {
-          strategy: 'always',
+        startDate: {
```

Comment:
This is the wrong place for that kind of configuration. effectivity dates belong to a whole theme and not to a component.
But if you scope it to @shopgate/engage/product/EffectivityDates it tell everyone else that it belongs to a widget.